src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -83,9 +86,10 @@ class BankAccountStep extends React.Component {
         return errors;
     }
 
-    validatePlaidAccount(values) {
+    validatePlaidAccount() {
```

Comment:
I think we can skip validating Plaid account, as we don't allow the user to continue unless they choose an account. I don't see a way of triggering the validation error if it exists. @luacmartins what do you think?

Reply:
Yes I think we can skip this, removed validatePlaidAccount  for now

Reply:
Makes sense since we can't display that error and the user can't continue without selecting an account.